content/ADR-215-gltfcomponent-sdk-component.md

```diff
@@ -122,30 +122,16 @@ Each entity will have its own animation state. The engine MUST provide a way to
 
 This version of the component does not specify any way to modify the materials of the models. This is a future work.
 
-## Handling colliders meshes
-
-### Handling `_collider` meshes
-
-All meshes of a node with a name ending in `_collider` will generate internally `MeshCollider`. Those meshes MUST be invisible and the
-rest of the properties of the mesh like its position in an animation should be honored. That enables colliders to be animated along with visible meshes. It is RECOMMENDED that all the engines implement a way to visualize the colliders for debugging purposes, making these meshes visible with a distinctive material.
-
-This convention is permanent and is not affected by any flag or configuration.
-
-### Automatic physics colliders
+## Handling visibility & colliders
 
-Physics colliders (`ColliderMask.CL_PHYSICS`) are automatically generated based on the `_collider` meshes if the property `disable_physics_colliders == false`.
+All meshes belong either to visible or invisible category. Names of invisible meshes should end in `_collider`. Those meshes MUST be invisible and the
+rest of the properties of the mesh like its position in an animation should be honored. This convention is permanent and is not affected by any flag or configuration.
 
-It is possible that the value of `disable_physics_colliders` is changed at runtime, so the engine SHOULD be able to enable/disable the colliders at runtime.
+Collision layers of both visible and invisible meshes are set via `visible_meshes_collision_mask` and `invisible_meshes_collision_mask`. Collision mask set to 0 indicates that collision detection / physics system should ignore meshes of this group, otherwise an engine MUST generate a collider for each mesh of this group and set its collision layers accordingly.
 
-Skinned meshes are not supported for colliders of any kind.
-
-### Automatic pointer colliders
-
-All visible meshes (not ending in `_collider`) MAY create an internal `MeshCollider` if the property `create_pointer_colliders == true`. This is useful for meshes that are not meant to be colliders but that should still be able to receive pointer events. Those colliders will have the `ColliderMask.CL_POINTER` mask.
+By default, collision mask of visible meshes is set to 0, but it could be set, for example, to CL_POINTER if it is desirable for pointer meshes to receive pointer events.
```

Comment:
```suggestion
By default, collision mask of visible meshes is set to 0, but it could be set, for example, to `CL_POINTER` if it is desirable for pointer meshes to receive pointer events.
```